### PR TITLE
Fix shell check

### PR DESCRIPTION
**Pull request by clarklee92**
Signed-off-by: clarklee92 <clarklee1992@hotmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix reference:
https://github.com/koalaman/shellcheck/wiki/SC2068
https://github.com/koalaman/shellcheck/wiki/SC2086
https://github.com/koalaman/shellcheck/wiki/SC2164
**Which issue(s) this PR fixes**:

>Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: third_party/multiarch/qemu-user-static/register/register.sh [hunk 1]
```diff
@@ -19,13 +19,13 @@ entries="aarch64 aarch64_be alpha arm armeb hppa m68k microblaze microblazeel mi
 if [ "${1}" = "--reset" ]; then
     shift
     (
-    cd /proc/sys/fs/binfmt_misc
+    cd /proc/sys/fs/binfmt_misc || exit
     for file in $entries; do
-        if [ -f qemu-${file} ]; then
-            echo -1 > qemu-${file}
+        if [ -f qemu-"${file}" ]; then
```
Comment (BenTheElder) [inline third_party/multiarch/qemu-user-static/register/register.sh]: style nit: we try to prefer quoting the whole string in our scripts:
```
        if [ -f "qemu-${file}" ]; then
```

**Comment by BenTheElder** [comment]
generally LGTM, one nit

**Comment by clarklee92** [comment]
> generally LGTM, one nit

Thank you😬, I already updated.

File: third_party/multiarch/qemu-user-static/register/register.sh [hunk 2]
```diff
@@ -19,13 +19,13 @@ entries="aarch64 aarch64_be alpha arm armeb hppa m68k microblaze microblazeel mi
 if [ "${1}" = "--reset" ]; then
     shift
     (
-    cd /proc/sys/fs/binfmt_misc
+    cd /proc/sys/fs/binfmt_misc || exit
```
Comment (liggitt) [inline third_party/multiarch/qemu-user-static/register/register.sh]: do we normally make edits like this under third_party?

Comment (BenTheElder) [reply third_party/multiarch/qemu-user-static/register/register.sh]: for this one at least, yes https://github.com/kubernetes/kubernetes/pull/69820

Comment (BenTheElder) [reply third_party/multiarch/qemu-user-static/register/register.sh]: for the go code we usually have edited code under third_party/forked, but I don't think we've been fully consistent about things being in `third_party/forked` vs `third_party` ...

**Comment by clarklee92** [comment]
/assign @cjcullen @lavalamp
Need your `/approve` if the change is ok ):

**Comment by lavalamp** [comment]
Sorry, I didn't see this until now.

/approve

**Pull request merged**